packages/server/graphql/intranetSchema/mutations/updateEmail.ts

```diff
@@ -0,0 +1,74 @@
+import {GraphQLBoolean, GraphQLNonNull, GraphQLString} from 'graphql'
+import getRethink from '../../../database/rethinkDriver'
+import {requireSU} from '../../../utils/authorization'
+import {InternalContext} from '../../graphql'
+import updateUser from '../../../postgres/queries/updateUser'
+
+const updateEmail = {
+  type: GraphQLNonNull(GraphQLBoolean),
+  description: `Updates the user email`,
+  args: {
+    oldEmail: {
+      type: GraphQLNonNull(GraphQLString),
```

Comment:
if you change this to a GraphQLID then graphql will validate that it doesn't contain any spaces

Reply:
@mattkrick I see we have https://github.com/ParabolInc/parabol/blob/8dbff9b4c6d19467c96b323be63f61e2926dc0a5/packages/server/graphql/types/GraphQLEmailType.ts, maybe that's the best of both worlds? I see it's not trimming the value but I can add that and then it'd be consistent across the codebase. Also, all the checks would be explicit which I think is a bit more readable. What do you think?

Reply:
oh yeaaah forgot about that type! no strong preference either way, whatever you think looks best

Reply:
Changed to `GraphQLEmailType`. I think it should be consistent across the codebase.